plugins/modules/ecs_service.py

```diff
@@ -195,23 +194,32 @@
         required: false
         choices: ["DAEMON", "REPLICA"]
         type: str
+    propagate_tags:
+        description:
+          - Propagate tags from ECS task defintition or ECS service to ECS task.
+        required: false
+        choices: ["TASK_DEFINITION", "SERVICE"]
+        type: str
+    tags:
+        description:
+          - A dictionary of tags to add or remove from the resource.
+        type: dict
+        required: false
+
```

Comment:
I would also suggest to add `purge_tags` and handle tags update.

Reply:
I dont think its the right place for this, this module created and delete services.
to manipulate tags we have ecs_tag module.

Reply:
Yeah, I guess you are right, this is managed by the ecs_tag module.